recipes/sortmerna/4.2.0/meta.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# FILE: meta.yaml
+# Created: Dec 08, 2019 Sun
+#
+{% set version = '4.2.0' %}
+{% set md5 = 'e684ba3893b9bc818f0763941ce1700c' %}
+
+package:
+  name: sortmerna
+  version: {{version}}
+  
+# TODO:
+#   - for now using pre-built binaries due complexities with C++17 and the dependencies.
+
+source:
+  url: https://github.com/biocore/sortmerna/releases/download/v{{version}}/sortmerna-4.2.0-Linux.tar.gz
+  md5: {{md5}}
```

Comment:
Also, please write the checksum here directly (and don’t `{% set %}` it). It’s recommended to introduce extra Jinja2 variables only if they are actually used more than once in the recipe.

Reply:
done